src/main/java/seedu/address/ui/DrugCard.java

```diff
@@ -29,16 +29,6 @@ public class DrugCard extends UiPart<Region> {
     private Label tradeName;
     @FXML
     private Label id;
-    @FXML
-    private Label activeIngredient;
-    @FXML
-    private Label direction;
-    @FXML
-    private Label purposes;
-    @FXML
-    private Label sideEffects;
-    @FXML
-    private Label storageCount;
```

Comment:
Great, we can definitely start reorganising the dense text information for the Drug Card.